### ISSUE TITLE
Ability for UQL to understand chained properties in addition to classes

### ISSUE THREAD

**Issue by kbower** [kbower/tickettest1#1329]
_Imported from trac ticket 1329.
Created by **kbower**
Opened in trac: Jan 3, 2012, 11:37:00 AM
Last modified in trac: Jan 3, 2012, 12:17:46 PM_
***
Example:
```
https://dev/tg/order/query?q=secondarysalesperson.User.firstname-SWKen
```

**Comment by kbower** [comment]
_**kbower** trac comment on Jan 3, 2012, 11:37:48 AM:_

(Changeset [[2098]](../commit/7e7dbc8463c2daf2c3b17671904e88251b3ada68))
 * Ability for UQL to understand chained properties in addition to classes (closes #1329)

**Comment by kbower** [comment]
_**kbower** trac status change on Jan 3, 2012, 11:37:48 AM:_ **closed**

**Comment by kbower** [comment]
_**kbower** trac status change on Jan 3, 2012, 11:50:04 AM:_ **reopened**

**Comment by kbower** [comment]
_**kbower** trac comment on Jan 3, 2012, 11:50:51 AM:_

UQL should not fail for
```
https://dev/tg/order/query?q=secondarysalesperson.user.firstname-SWKen
```

**Comment by kbower** [comment]
_**kbower** trac comment on Jan 3, 2012, 12:17:46 PM:_

(Changeset [[2099]](../commit/6e49ff281d502424b54577d3c2c95bcd6943d090))
 * UQL didn't understand chained attributes, such as `http://dev:8500/order/query?q=primarysalesperson.user.firstname-SWKen` (fixes #1329)

**Comment by kbower** [comment]
_**kbower** trac status change on Jan 3, 2012, 12:17:46 PM:_ **closed**
